pdp_util/raster.py

```diff
@@ -238,22 +239,80 @@ def __call__(self, environ, start_response):
         return response_200(start_response, content)
 
 
-def build_orca_url(handlers, thredds_root, req):
+def build_orca_url(handlers, orca_root, req):
     """orca is the OPeNDAP Request Compiler Application which pulls apart large OPeNDAP requests
     to THREDDS into bite-sized chunks and then reasemmbles them for the user.
 
-    Orca is available through a url with the format:
-    [thredds_root]/[filepath]:[variable][time_start:time_end][lat_start:lat_end][lon_start:lon_end]
+    Orca is available through a url with one of the formats:
+    1. [orca_root]/?filepath=[filepath]
+    2. [orca_root]/?filepath=[filepath]&targets=time[time_start:time_end],lat[lat_start:lat_end],lon[lon_start:lon_end],[variable][time_start:time_end][lat_start:lat_end][lon_start:lon_end]
 
     where the [filepath] can be attained by the mapping of handler url to handler file from a config dict
     """
     filename = None
```

Comment:
Should this inline computation be replaced by a call to `get_filepath_from_handlers` below?